devstack/plugin.sh

```diff
@@ -20,8 +20,22 @@ setup_stack()
 
 		"post-config")
 			source "$NEUTRON_DIR/devstack/plugin.sh"
-			source "$NEUTRON_DIR/devstack/lib/ml2" contrail_driver
-			enable_ml2_extension_driver
+
+			# Insert extension driver
```

Comment:
This looks to me very similar as configure_plugin() function from install_functions.sh file. Please avoid code duplication.